README.md

```diff
@@ -144,7 +144,7 @@ it is possible to use other tools that create the expected certificates for Isti
 iop istio-system istio-system-security $IBASE/istio-system-security
 ```
 
-Important options: the 'dnsCerts' list allows associating DNS certs with specific service accounts.
+**Important options**: the 'dnsCerts' list allows associating DNS certs with specific service accounts.
 This should be used if you plan to use Galley or Sidecar injector in different namespaces.
 By default it supports "istio-control", "istio-master" namespaces used in the examples.
```

Comment:
Would 'istio-control-master' not be the better name? It would be more consistent with the naming or are we on the wrong track?

Reply:
Sounds good - happy to merge a pull request ! 
Or maybe 'istio-control-canary' ? Since the purpose is to test the next version, master happens to be the branch but we may switch to 'master is stable'.